pkg/network/master/master.go

```diff
@@ -231,6 +232,8 @@ func clusterNetworkChanged(obj *networkapi.ClusterNetwork, old *networkapi.Clust
 		return true, nil
 	} else if len(old.ClusterNetworks) != len(obj.ClusterNetworks) {
 		return true, nil
+	} else if old.VXLANPort != obj.VXLANPort {
```

Comment:
This is comparing two pointers but it needs to compare their values.

Also, I don't think we want to make this an error case... This is like expanding the clusterNetworkCIDR; yes, you have to restart all of the nodes too, but once you do, the nodes will deal with the change, so we should allow it.